pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/JobConfigConstants.java

```diff
@@ -19,6 +19,7 @@
 public class JobConfigConstants {
   public static final String PATH_TO_INPUT = "path.to.input";
   public static final String PATH_TO_OUTPUT = "path.to.output";
+  public static final String PATH_TO_SCHEMA = "path.to.schema";
```

Comment:
Can we put a comment for this one that we are deprecating this and prefer that they fetch it from the controller?